hack/validate/golangci-lint.yml

```diff
@@ -23,10 +23,8 @@ linters:
       - docs
 
     skip-files:
+      # TODO re-generate .proto files to improve detection as auto-generated code.
```

Comment:
Looks like gogo/protobuf was updated in https://github.com/gogo/protobuf/commit/9aee2206303e7734a27dc7fafbd6f9668a4b9ae5 to match the correct format, but I found `.pb.go` files that were generated with an older version, e.g. https://github.com/moby/moby/blob/7a7357dae1bcccb17e9b2d4c7c8f5c025fce56ca/builder/remotecontext/tarsum.pb.go#L1-L3


We should have makefile targets to regenerate those files

Reply:
can we regenerate in this PR? Or is there another PR for this?

Reply:
Have not yet had time to look at how they were generated, and what's needed. I prefer to do that in a separate PR as it's less trivial (and we'll have to write scripts, and likely steps in CI to validate)

Reply:
Opened https://github.com/moby/moby/issues/40176 as a tracking issue